### ISSUE TITLE
Missing dependency for python-ldap

### ISSUE THREAD

**Issue by jallakim** [digitalocean/netbox#371]
`python-ldap` is missing as a dependency in the LDAP documentation. It causes the `ImportError` exception when trying to load the LDAP configuration file (due to `import ldap`), but is never printed to logs or anything (even when `DEBUG = True`). Silently falls back to only using the users stored in the database.

**Comment by Gelob** [comment]
python-ldap is a dependency of django-auth-ldap so this shouldn't be needed.

```
root@quick-test:~# sudo pip install django-auth-ldap
Collecting django-auth-ldap
  Downloading django_auth_ldap-1.2.8-py2-none-any.whl
Collecting python-ldap>=2.0 (from django-auth-ldap)
  Downloading python-ldap-2.4.25.tar.gz (121kB)
    100% |████████████████████████████████| 122kB 3.4MB/s
Collecting django (from django-auth-ldap)
  Downloading Django-1.9.8-py2.py3-none-any.whl (6.6MB)
    100% |████████████████████████████████| 6.6MB 102kB/s
Requirement already satisfied (use --upgrade to upgrade): setuptools in /usr/lib/python2.7/dist-packages (from python-ldap>=2.0->django-auth-ldap)
Building wheels for collected packages: python-ldap
  Running setup.py bdist_wheel for python-ldap ... done
  Stored in directory: /root/.cache/pip/wheels/d2/29/f0/64c22b9fdfb77286ce1fad9837eb0388e704d2159911cc0847
Successfully built python-ldap
Installing collected packages: python-ldap, django, django-auth-ldap
Successfully installed django-1.9.8 django-auth-ldap-1.2.8 python-ldap-2.4.25.
```

**Comment by jallakim** [comment]
Hm, weird. I ran that command at some point during my installation, without failing, and I had to manually install `python-ldap`. Tried on a clean install now, and indeed, it pulls `python-ldap` as it did in your test.

It might be a one-off, but it did in fact happen. Digging in `~/.bash_history`, these are the steps done during the initial install of Netbox;

```
pip install -r requirements.txt
pip install --upgrade cffi
pip install -r requirements.txt
pip install django-auth-ldap
apt-get install -y python2.7 python-dev git python-pip libxml2-dev libxslt1-dev libffi-dev graphviz libpq-dev python-dev libldap2-dev libsasl2-dev libssl-dev
pip install django-auth-ldap
pip install --upgrade django-auth-ldap
```

So it might be related to the fact that I tried to install `django-auth-ldap` before having the needed libraries. Trying that now on a clean install, it still pulls the `python-ldap` module.

I don't mind not updating the documentation, but it doesn't hurt imho.
